fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -78,10 +77,10 @@ class InterpolatedThresholder(BaseEstimator, MetaEstimatorMixin):
        [Online]. Available: https://arxiv.org/abs/1610.02413.
     """
 
-    def __init__(self, estimator, interpolation_dict, prefit=False,
+    def __init__(self, estimator, threshold_interpolation, prefit=False,
                  predict_method='deprecated'):
         self.estimator = estimator
-        self.interpolation_dict = interpolation_dict
+        self.threshold_interpolation_ = threshold_interpolation
```

Comment:
this looks like a meta-estimator to me, so I'd rather it follow estimator API and have this as `threshold_interpolation` instead of the one with a trailing underscore.

Reply:
It is indeed a meta-estimator. Changed it in the latest commit (which goes to a separate PR), but I have to say I wasn't aware of different rules for meta-estimators vs estimators. If you happen to have a reference I'd love to educate myself 🙂

Reply:
I meant "it's an estimator because it's a meta-estimator." All meta-estimators are estimators, and there isn't much to it other than: 

https://github.com/scikit-learn/scikit-learn/blob/45bb9ab916351c95a6d15a5cb06469036a0707cf/sklearn/base.py#L810-L813

Reply:
In more advanced cases, there are things such as _estimator tag delegation_ and passing `fit_params` and `score_params` to the underlying estimator, etc, but that can come next.